loader/checkpoint.go

```diff
@@ -395,15 +395,23 @@ func (cp *RemoteCheckPoint) GenSQL(filename string, offset int64) string {
 }
 
 // UpdateOffset implements CheckPoint.UpdateOffset.
-func (cp *RemoteCheckPoint) UpdateOffset(filename string, offset int64) {
+func (cp *RemoteCheckPoint) UpdateOffset(filename string, offset int64) error {
 	cp.restoringFiles.Lock()
 	defer cp.restoringFiles.Unlock()
 	db, table, err := getDBAndTableFromFilename(filename)
 	if err != nil {
-		cp.logger.Error("error in checkpoint UpdateOffset", zap.Error(err))
-		return
+		return terror.ErrLoadTaskCheckPointNotMatch.Generate()
```

Comment:
It's different error here.

Reply:
i think's it's a same error that  is an error caused by an incorrect filename？

Reply:
How about use terror.Annotate?

Reply:
you mean annotate two error with different  error msg ?